docs/getstarted/evaluation.md

```diff
@@ -1,36 +1,30 @@
 (get-started-evaluation)=
-# Evaluation
+# Evaluating Your Test Set
 
-Welcome to the ragas quickstart. We're going to get you up and running with ragas as quickly as you can so that you can go back to improving your Retrieval Augmented Generation pipelines while this library makes sure your changes are improving your entire pipeline.
+Once your test set is ready (whether you've created your own or used the [synthetic test set generation module](get-started-testset-generation)), it's time to evaluate your RAG pipeline. Our aim is to help you set up with Ragas as quickly as possible so that you can focus on enhancing your Retrieval Augmented Generation pipelines while this library ensures your changes are improving the entire pipeline.
 
-to kick things of lets start with the data
-
-:::{note}
-Are you using Azure OpenAI endpoints? Then checkout [this quickstart
-guide](../howtos/customisations/azure-openai.ipynb)
-:::
-
-```bash
-pip install ragas
-```
-
-Ragas also uses OpenAI for running some metrics so make sure you have your openai key ready and available in your environment
+This guide uses OpenAI for running some metrics, so make sure you have your OpenAI key ready and available in your environment.
 
 ```python
 import os
 os.environ["OPENAI_API_KEY"] = "your-openai-key"
 ```
+:::{note}
+By default, these metrics use OpenAI's API to compute the score. If you're using this metric, ensure that you've set the environment key `OPENAI_API_KEY` with your API key. You can also try other LLMs for evaluation, check the [LLM guide](../howtos/customisations/llms.ipynb) to learn more.
+:::
+
+Let's start with the data.
+
 ## The Data
 
-For this tutorial we are going to use an example dataset from one of the baselines we created for the [Financial Opinion Mining and Question Answering (fiqa) Dataset](https://sites.google.com/view/fiqa/). The dataset has the following columns.
+For this tutorial, we'll use an example dataset from one of the baselines we created for the [Amnesty QA](https://huggingface.co/datasets/explodinggradients/amnesty_qa) dataset. The dataset contains the following columns:
 
 - question: `list[str]` - These are the questions your RAG pipeline will be evaluated on.
-- answer: `list[str]` - The answer generated from the RAG pipeline and given to the user.
+- answer: `list[str]` - The answer generated from the RAG pipeline and provided to the user.
 - contexts: `list[list[str]]` - The contexts which were passed into the LLM to answer the question.
-- ground_truths: `list[list[str]]` - The ground truth answer to the questions. (only required if you are using context_recall)
-
-Ideally your list of questions should reflect the questions your users give, including those that you have been problematic in the past.
+- ground_truth: `list[str]` - The ground truth answer to the questions.
 
+Ideally, your list of questions should reflect the questions your users ask, including those that have been problematic in the past.
```

Comment:
> Ideally, your list of questions should reflect the questions your users ask, including those that have been problematic in the past.

Wbt
An ideal test data set should contain samples as close as the ones expected in your real-world use case.